test/integration/targets/win_unzip/tasks/tests.yml

```diff
@@ -0,0 +1,62 @@
+- name: Unarchive sysinternals archive
+  win_unzip:
+    src: C:\Windows\Temp\SysinternalsSuite.zip
+    dest: C:\Program Files\sysinternals
+    #recurse: yes
+  register: unzip_achive
+
+- name: Test unzip_archive
+  assert:
+    that:
+    - unzip_achive|changed
+
+
+- name: Unarchive sysinternals archive again
+  win_unzip:
+    src: C:\Windows\Temp\SysinternalsSuite.zip
+    dest: C:\Program Files\sysinternals
+    #recurse: yes
```

Comment:
Nope, it was just a reminder that we don't have an example where we have an archive inside another archive.